attestation-service/verifier/Cargo.toml

```diff
@@ -30,6 +31,7 @@ csv-rs = { git = "https://github.com/openanolis/csv-rs", rev = "b74aa8c", option
 eventlog-rs = { version = "0.1.3", optional = true }
 hex.workspace = true
 jsonwebtoken = { workspace = true, default-features = false, optional = true }
+jsonwebkey = "0.3.5"
```

Comment:
we should share the jsonwebtoken crate in the workspace

Reply:
jsonwebtoken  or jsonwebkey ?

Reply:
ah, jsonwebkey. but do we need to add this dependency in this PR at all?

Reply:
Its not required now, this dependency already included and merged in other PR